sdk/lib/rtl/unicode.c

```diff
@@ -1367,7 +1367,7 @@ RtlIsTextUnicode(CONST VOID* buf, INT len, INT* pf)
             }
         }
 
-        if (lead_byte)
+        if (lead_byte && pf && (*pf & IS_TEXT_UNICODE_DBCS_LEADBYTE))
```

Comment:
Moved conditions can move to line 1359 even.

Reply:
Those need to be done only on lead_byte.

Reply:
> Those need to be done only on lead_byte.

PR code looks the opposite to me : `lead_byte` is useless unless the flag is present.

Reply:
Partially moved.